recipes/ginkgo/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.9)
+project(test_install)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(test_install test_install.cpp)
+target_link_libraries(test_install ${CONAN_LIBS})
```

Comment:
It's best if you tested the components you declared

 https://github.com/conan-io/conan-center-index/blob/0695c9d81f94a9a2ee9ecf25b82dab7a19ea38eb/docs/reviewing.md#verifying-components